public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -63,6 +113,36 @@ export const ConfigEditor: FC<Props> = (props: Props) => {
             title={'The timeout must be a valid duration string, such as "15m" "30s" "2000ms" etc.'}
           />
         </InlineField>
+        <InlineField
+          label="Default Log Groups"
+          labelWidth={28}
+          tooltip="Optionally, specify default log groups for CloudWatch Logs queries."
+        >
+          <MultiSelect
```

Comment:
You could also move this to its own component that is imported here and in the LogsQueryField. Then there would be no need to import constants such as MAX_LOG_GROUPS and MAX_VISIBLE_LOG_GROUPS.

Reply:
done